app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+class User < ActiveRecord::Base
+  def self.find_or_create_from_auth(auth)
```

Comment:
Assignment Branch Condition size for find_or_create_from_auth is too high. [36.01/15]